Code/Tools/Libs/GuiFoundation/QtResources/resources.qrc

```diff
@@ -1,7 +1,7 @@
 <RCC>
   <qresource prefix="GuiFoundation">
-	<file>EZ-logo.svg</file>
-	<file>Icons/Add16.png</file>
+    <file>EZ-logo.svg</file>
```

Comment:
Where are the SVGs from? Do we need to mention some license?

Reply:
I made them myself